docs/docsite/rst/user_guide/intro_patterns.rst

```diff
@@ -153,6 +153,10 @@ Finally, you can use ``--limit`` to read the list of hosts from a file by prefix
 
     ansible-playbook site.yml --limit @retry_hosts.txt
 
+If ``RETRY_FILES_ENABLED`` is set to ``True``, a ``.retry`` file will be created after the ``ansible-playbook`` run containing a list of failed hosts from all plays. This file is overwritten each time ``ansible-playook`` is finishing running.
```

Comment:
Is there a way to link to the [this HTML page](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#retry-files-enabled) from within the RST?

Reply:
Yes. If you look at the reference_appendices/config.rst file (if you've done a make webdocs locally it should be there), you'll see every option is preceded by an .rst anchor that matches the  option name. So in this instance, :ref:`RETRY_FILES_ENABLED` should work.

Reply:
That's perfect. Thanks!